includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -360,35 +384,50 @@ public static function display_invalid_url_validation_error_counts_summary( $pos
 
 		$result = array();
 		if ( $counts['new'] ) {
-			if ( AMP_Validation_Manager::is_sanitization_forcibly_accepted() ) {
-				$icon = 'flag';
```

Comment:
Are we still using something like a flag icon? Maybe deleting this was premature.